mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java

```diff
@@ -5,6 +5,7 @@
 
 package com.mifos.mifosxdroid.core;
 
+
```

Comment:
What I meant by unnecessary blank lines were these - added by you and not those which Android Studio automatically inserts to distinguish imports from different packages. Please keep the blank lines between the imports as they were.

Reply:
done!